From ee290e35c3237a43b96b12aa4eeb18dec388becc Mon Sep 17 00:00:00 2001
From: Murali Karicheri <m-karicheri2@ti.com>
Date: Wed, 4 Mar 2020 23:33:31 +0000
Subject: [PATCH 17/28] net: ti: icssg-prueth: use constants instead of
 hardcoded values

Use existing constants for speed and duplex values instead of
hardcoded values. While at it also remove the unnecessary check
for 100M since default value of gig_en is false.

Signed-off-by: Murali Karicheri <m-karicheri2@ti.com>
Acked-by: Roger Quadros <rogerq@ti.com>
---
 drivers/net/ti/icssg-prueth.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/drivers/net/ti/icssg-prueth.c b/drivers/net/ti/icssg-prueth.c
index e61dc51247..971a4631f1 100644
--- a/drivers/net/ti/icssg-prueth.c
+++ b/drivers/net/ti/icssg-prueth.c
@@ -254,12 +254,10 @@ static int icssg_update_link(struct prueth *priv)
 	bool gig_en = false, full_duplex = false;
 
 	if (phy->link) { /* link up */
-		if (phy->speed == 1000)
+		if (phy->speed == SPEED_1000)
 			gig_en = true;
-		if (phy->duplex == 0x1)
+		if (phy->duplex == DUPLEX_FULL)
 			full_duplex = true;
-		if (phy->speed == 100)
-			gig_en = false;
 		/* Set the RGMII cfg for gig en and full duplex */
 		icssg_update_rgmii_cfg(priv->miig_rt, gig_en, full_duplex,
 				       priv->slice);
-- 
2.25.1

